packages/backend/src/apps/gathersg/actions/get-case-details/get-data-out-metadata.ts

```diff
@@ -0,0 +1,127 @@
+import { IDataOutMetadata, IExecutionStep, IJSONArray } from '@plumber/types'
+
+import { dataOutSchema } from './schema'
+
+async function getDataOutMetadata(
+  step: IExecutionStep,
+): Promise<IDataOutMetadata> {
+  const { dataOut: rawDataOut } = step
+  if (!rawDataOut) {
+    return null
+  }
+
+  const parsedDataOut = dataOutSchema.safeParse(rawDataOut)
+  if (!parsedDataOut.success) {
+    return null
+  }
+
+  const { data: dataOut } = parsedDataOut.data
+
+  const metadata = {
+    data: {
+      type: {
+        name: { label: 'Case type' },
+        uuid: { isHidden: true },
+      },
+      uuid: { label: 'Case UUID' },
+      caseRef: { label: 'Case ref' },
+      source: { label: 'Case source' },
+      status: {
+        name: { label: 'Status' },
+        uuid: { isHidden: true },
+        color: { isHidden: true },
+        isFinal: { isHidden: true },
+      },
+      createdAt: { label: 'Created at' },
+      createdBy: {
+        name: { label: 'Created by (name)' },
+        email: { label: 'Created by (email)' },
+        role: { isHidden: true },
+        uuid: { isHidden: true },
+      },
+      updatedAt: { label: 'Updated at' },
+      updatedBy: {
+        name: { label: 'Updated by (name)' },
+        email: { label: 'Updated by (email)' },
+        role: { isHidden: true },
+        uuid: { isHidden: true },
+      },
+      durationSec: { isHidden: true },
```

Comment:
not sure if user will want to know how long has the case duration been ongoing: if yes we prob can just take the seconds and convert to days?